schedule/yast/sle_15-sp4/defaults/default_full.yaml

```diff
@@ -0,0 +1,8 @@
+---
+<<: !include schedule/yast/sle_15-sp4/defaults/default.yaml
+registration:
+  - installation/registration/skip_registration
+extension_module_selection:
+  - installation/module_selection/skip_module_selection
+add_on_product:
```

Comment:
I would be more explicit here:
`add_on_product: []` is not processed and now 
```
add_on_product_installation: 
 - installation/add_on_product_installation/accept_add_on_installation
 ```
 takes over, which is [a different screen](https://openqa.suse.de/tests/8344537#step/accept_add_on_installation/1)
 for which we need a hook in `default.yaml`